priv/rel/files/boot

```diff
@@ -139,8 +139,10 @@ generate_vm_args() {
     if [ "$COOKIE_MODE" = "ignore" ]; then
         printf "Cookie will be used from \"$HOME/.erlang.cookie\".\n"
         sed -i -e '/-setcookie/s/^/## removed: /' $VMARGS_PATH
-        COOKIE=`cat $HOME/.erlang.cookie`
-        printf "\n## set due to \"COOKIE_MODE=ignore\"\n-setcookie $COOKIE\n" >> $VMARGS_PATH
+        if [ -s $HOME/.erlang.cookie ] ; then
```

Comment:
"ignore" means ignore. we will not read from $HOME, we will not change cookie in vmargs!